package.json

```diff
@@ -44,6 +44,7 @@
     "prop-types": "15.8.1",
     "react": "16.14.0",
     "react-dom": "16.14.0",
+    "react-intl": "^5.25.1",
```

Comment:
Check this version in the Learning MFE. @sergivalero20

Reply:
@Squirrel18 With the current version doesn't cause any conflict. I run `npm run build` and everything is fine.